test-case/check-s3.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
+
+##
+## Case Name: check S3 status
```

Comment:
please make the name into `check-suspend-resume`
Not all platform are using S3 now.